### DISCUSSION TITLE
Missing Shimmer

### DISCUSSION THREAD

**Discussion by MarvAUT** [iotaledger/firefly#7642] [Help & Support]
Hi all, today I had a look into my Firefly Wallet and recognized, that my Shimmers aren't visible. So I did the update to Firefly 2.02 and they are still not there. Any ideas? thx.

**Comment by begonaalvarezd** [comment]
Quick information about SMR: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

Will close the issue because the new firefly app is not intended to display old staking rewards, shimmer tokens should be managed from the shimmer app. 

Please let us know if you have any other issues
